tests/DCPS/XTypes/run_test.pl

```diff
@@ -0,0 +1,66 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+     & eval 'exec perl -S $0 $argv:q'
+     if 0;
+
+use lib "$ENV{ACE_ROOT}/bin";
+use lib "$ENV{DDS_ROOT}/bin";
+use PerlDDS::Run_Test;
+use Getopt::Long;
+use strict;
+
+my $test = new PerlDDS::TestFramework();
+$test->enable_console_logging();
+
+my $verbose = 0;
+my $test_name = "";
+GetOptions(
+  "verbose" => \$verbose,
+  "test|f=s"   => \$test_name,
+);
+
+my @common_args = ('-DCPSConfigFile rtps_disc.ini -ORBDebugLevel 1 -DCPSDebugLevel 6');
+if ($verbose) {
+  push(@common_args, "--verbose");
+}
+
+my %params = (
+  "FirstTest"                             => {reader_type => "Property_1", writer_type => "Property_1", expect_to_fail => ""},
```

Comment:
Remove some of the space between the keys and the values. Also maybe make `expected_to_fail` a boolean (0 or 1 in Perl).

Reply:
Will do later, when all tests are in the list and I know how long is the longest name.